### ISSUE TITLE
Logging in while registering for marathon match drops url param

### ISSUE THREAD

**Issue by ajefts** [topcoder-platform/tc-website#11]
Try to register for a marathon match if  you're not logged in.  It's dropping the rd= param.  An example URL is https://community.topcoder.com/tc?module=MatchDetails&rd=16996.

After you login you end up at https://community.topcoder.com/tc?module=MatchDetails

**Comment by ajefts** [comment]
@skyhit think we could run a F2F to figure this out?

**Comment by skyhit** [comment]
are we using this one for marathon match - https://github.com/appirio-tech/arena-longcontest?

**Comment by ajefts** [comment]
No, I believe it is https://github.com/topcoder-platform/tc-website/tree/dev/src/main/com/topcoder/web/codinginterface/longcontest

I think it's the way it's redirecting to the login. If we get it to accounts.topcoder.com I think we can handle the retUrl correctly without dropping params...

**Comment by skyhit** [comment]
@ajefts as I know we split before (long time ago), maybe we switch back to use tc-website again.

**Comment by skyhit** [comment]
@ajefts I tried, but after registration, I was ended up at https://community.topcoder.com/longcontest/?module=ViewActiveContests&rt=13

which is pretty good.

**Comment by ajefts** [comment]
Thanks for looking. We just figured out a fix so we are ok on this one now.
